clar2wasm/tests/wasm-generation/noop.rs

```diff
@@ -0,0 +1,27 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::{int, uint, PropValue};
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_noop_to_int(val in uint().prop_map(PropValue)) {
+        crosscheck_compare_only(
+            &format!("(to-int {val})")
+        )
+    }
+}
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_noop_to_uint(val in int().prop_map(PropValue)) {
+        crosscheck_compare_only(
+            &format!("(to-uint {val})")
+        )
```

Comment:
Same remark.

Reply:
The intention on this test is to direct match both compiler and interpreter without any other step. We can create other more complex property test. What you think?

Reply:
same answer as above